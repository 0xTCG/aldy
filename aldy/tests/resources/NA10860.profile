CFTR:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e14: [0]
  e15: [0]
  e16: [0]
  e17: [0]
  e18: [0]
  e19: [0]
  e2: [0]
  e20: [0]
  e21: [0]
  e22: [0]
  e23: [0]
  e24: [0]
  e25: [0]
  e26: [0]
  e27: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i13: [0]
  i14: [0]
  i15: [0]
  i16: [0]
  i17: [0]
  i18: [0]
  i19: [0]
  i2: [0]
  i20: [0]
  i21: [0]
  i22: [0]
  i23: [0]
  i24: [0]
  i25: [0]
  i26: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
COMT:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP1A1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP1A2:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2A13:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
CYP2A6:
  down: [0, 0]
  e1: [0, 0]
  e2: [0, 0]
  e3: [0, 0]
  e4: [0, 0]
  e5: [0, 0]
  e6: [0, 0]
  e7: [0, 0]
  e8: [0, 0]
  e9: [0, 0]
  i1: [0, 0]
  i2: [0, 0]
  i3: [0, 0]
  i4: [0, 0]
  i5: [0, 0]
  i6: [0, 0]
  i7: [0, 0]
  i8: [0, 0]
  up: [0, 0]
  utr3: [0, 0]
  utr5: [0, 0]
CYP2B6:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2C19:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2C8:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2C9:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2D6:
  e1: [5773, 4876]
  e2: [7614, 4775]
  e3: [5943, 4655]
  e4: [8663, 3342]
  e5: [8894, 5152]
  e6: [6165, 4395]
  e7: [11864, 2538]
  e8: [4121, 4153]
  e9: [5298, 6829]
  i1: [27006, 21449]
  i2: [23855, 13477]
  i3: [4053, 1914]
  i4: [24528, 9574]
  i5: [7952, 6407]
  i6: [9447, 5216]
  i7: [21544, 8546]
  i8: [2942, 3288]
  ins: [11289, 22153]
  pce: [0, 73016]
  rep: [82361, 101987]
  up: [231916, 142157]
  utr3: [1630, 2608]
  utr5: [4300, 2899]
CYP2E1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  utr3: [0]
  utr5: [0]
CYP2F1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2J2:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2R1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2S1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP2W1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP3A4:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP3A43:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
CYP3A5:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP3A7:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
CYP4F2:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
DPYD:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e14: [0]
  e15: [0]
  e16: [0]
  e17: [0]
  e18: [0]
  e19: [0]
  e2: [0]
  e20: [0]
  e21: [0]
  e22: [0]
  e23: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i13: [0]
  i14: [0]
  i15: [0]
  i16: [0]
  i17: [0]
  i18: [0]
  i19: [0]
  i2: [0]
  i20: [0]
  i21: [0]
  i22: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
G6PD:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
GSTM1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
GSTP1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
IFNL3:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
NAT1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  i1: [0]
  i2: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
NAT2:
  down: [0]
  e1: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
NUDT15:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  i1: [0]
  i2: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
SLCO1B1:
  down: [0]
  e1: [0]
  e10: [0]
  e11: [0]
  e12: [0]
  e13: [0]
  e14: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  e9: [0]
  i1: [0]
  i10: [0]
  i11: [0]
  i12: [0]
  i13: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  i8: [0]
  i9: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
TPMT:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  e7: [0]
  e8: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  i6: [0]
  i7: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
UGT1A1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
UGT2B7:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  e4: [0]
  e5: [0]
  e6: [0]
  i1: [0]
  i2: [0]
  i3: [0]
  i4: [0]
  i5: [0]
  up: [0]
VKORC1:
  down: [0]
  e1: [0]
  e2: [0]
  e3: [0]
  i1: [0]
  i2: [0]
  up: [0]
  utr3: [0]
  utr5: [0]
neutral:
  hg19: ['22', 42547463, 42548249]
  value: 19257

