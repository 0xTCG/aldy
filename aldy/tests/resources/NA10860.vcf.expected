##fileformat=VCFv4.2
##source=aldy-v4.0
##INFO=<ID=ANN,Number=1,Type=String,Description="Location within CYP2D6">
##INFO=<ID=TYPE,Number=1,Type=String,Description="Mutation kind">
##INFO=<ID=GENE,Number=1,Type=String,Description="Gene">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
##FORMAT=<ID=MA,Number=1,Type=String,Description="Major genotype star-allele calls">
##FORMAT=<ID=MI,Number=1,Type=String,Description="Minor genotype star-allele calls">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA10860:0:*1/*4+*4.021.ALDY	NA10860:1:*1/*4.021+*4N.ALDY
22	42522392	rs28371738	G	A	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:14:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:14:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42522613	rs1135840	C	G	0	PASS	EFFECT=S486T;GENE=CYP2D6	GT:DP:MA:MI	0|1|0:15:-,*4,-:-,*4.001,-	0|1|0:15:-,*4.021,-:-,*4.021,-
22	42523211	rs2004511	T	C	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:34:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:34:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42523409	rs1985842	T	G	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:59:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:59:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42523528	rs1058172	C	T	0	PASS	EFFECT=R365H;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:21:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:21:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42524696	rs2267447	T	C	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:34:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:34:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42524708	rs111564371	T	C	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	1|0|0:11:*1,-,-:*1.018,-,-	1|0|0:11:*1,-,-:*1.018,-,-
22	42524713	rs112568578	C	G	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	1|0|0:12:*1,-,-:*1.018,-,-	1|0|0:12:*1,-,-:*1.018,-,-
22	42524743	rs113889384	G	A	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	1|0|0:14:*1,-,-:*1.018,-,-	1|0|0:14:*1,-,-:*1.018,-,-
22	42524795	rs28371713	A	G	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	1|0|0:14:*1,-,-:*1.018,-,-	1|0|0:14:*1,-,-:*1.018,-,-
22	42524947	rs3892097	C	T	0	PASS	EFFECT=splice_defect;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:32:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:32:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42525132	rs1058164	C	G	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:24:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:24:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42525798	rs28371705	G	C	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:35:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:35:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42525811	rs28371704	T	C	0	PASS	EFFECT=H94R;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:34:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:34:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42525821	rs28371703	G	T	0	PASS	EFFECT=L91M;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:33:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:33:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42525952	rs28371702	A	C	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:23:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:23:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42526049	rs28371701	G	C	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	1|1|1:23:*1,*4,*4.021.ALDY:*1.018,*4.001,*4.1021	1|1|1:23:*1,*4.021,*4N.ALDY:*1.018,*4.021,*4.1013
22	42526484	rs28371699	C	A	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:31:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:31:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42526524	rs29001678	G	A	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	1|0|0:11:*1,-,-:*1.018,-,-	1|0|0:11:*1,-,-:*1.018,-,-
22	42526694	rs1065852	G	A	0	PASS	EFFECT=P34S;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:18:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:18:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42527793	rs1080989	C	T	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:30:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:30:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42528028	rs28735595	T	C	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:16:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:16:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
22	42528224	rs28588594	G	A	0	PASS	EFFECT=none;GENE=CYP2D6	GT:DP:MA:MI	0|1|1:30:-,*4,*4.021.ALDY:-,*4.001,*4.1021	0|1|1:30:-,*4.021,*4N.ALDY:-,*4.021,*4.1013
